content/api/rest-sdk/push.textile

```diff
@@ -0,0 +1,83 @@
+---
+title: Push
```

Comment:
@tbedford @m-hulbert can we please sync up on the page titles? I thought we were going to match the titles we decided on in the Miro board?

Reply:
I'm looking at this (which is linked to from the DR linked from the epic): 
![image](https://user-images.githubusercontent.com/25511700/163371500-5f1cad69-727c-4153-834d-43b226991f44.png)
Although we did decide to rename the Encryption object to Crypto as that is what the object actually is.

Reply:
We have to revisit the pages that we'd completed from before that meeting too, so I'm expecting we'll create a "unify titles" ticket to tidy these up once we've completed REST and realtime.

Reply:
I think you're looking at the conceptual part in that screenshot @tbedford - the API reference is further down.

Reply:
Quite right. Although the names used there more or less correspond. @Srushtika can we sync up on how we should be naming again please, thanks.